src/App.js

```diff
@@ -4,20 +4,20 @@ import { Card } from './components/Card';
 import './style.css';
 import logo from '../src/images/wifi.png';
 
+/* List of languages that require RTL direction (alphabetic order). */
+const RTL_LANGUAGES = ['ar', 'fa-IR'];
```

Comment:
this might just be my personal preference again but i think it would be better to have one locale per line (with trailing commas in each line) so that new entries would be just simple additions and no deletions.

Reply:
@abhinavxd That's my preference too! But I am bound by prettier here 🤔